README.md

```diff
@@ -294,7 +294,7 @@ How this works:
 // Set ASCII secret
 var secret = 'rNONHRni6BAk7y2TiKrv';
 
-// Get HOTP counter token at counter = 42 
```

Comment:
These superficial white space changes slow down the review process. I was thinking we could reformat the library using https://github.com/feross/standard in one commit and get it over with. Could you omit these whitespace changes for this commit?